pyston/order/managers.py

```diff
@@ -219,4 +220,8 @@ class DjangoOrderManager(BaseParserModelOrderManager, BaseDjangoOrderManager):
     """
 
     def _sort_queryset(self, qs, terms):
+        # Adding pk makes ordering deterministic, when column has multiple same values.
+        terms.append(
+            DjangoSorter(('pk',), DirectionSlug.DESC if terms[-1].descending else DirectionSlug.ASC).get_order_term()
+        )
```

Comment:
I think you should test, if PK is not already in `terms`. Also, could you write test for this, please?